addons/account/tests/test_account_journal_dashboard.py

```diff
@@ -96,40 +109,46 @@ def test_customer_invoice_dashboard(self):
         self.assertEqual(dashboard_data['number_late'], 2)
         self.assertIn('78.42', dashboard_data['sum_late'])
 
-    def test_sale_purchase_journal_for_multi_currency_purchase(self):
-        currency = self.currency_data['currency']
+    def test_sale_purchase_journal_for_purchase(self):
+        """
+        Test different purchase journal setups with or without multicurrency:
+            1) Journal in company currency, bills in foreign currency -> dashboard data should be displayed in company currency
+            2) Journal in foreign currency, bills in foreign currency -> dashboard data should be displayed in foreign currency
+            3) Journal in foreign currency, bills in company currency -> dashboard data should be displayed in foreign currency
+            4) Journal in company currency, bills in company currency -> dashboard data should be displayed in company currency
+            5) Journal in company currency, bills in foreign currency -> dashboard data should be displayed in company currency
```

Comment:
The last case is the same as the first one :)

Reply:
Good catch, but only the docstring is wrong, first case should be 
```
1) Journal with no currency, bills in foreign currency -> dashboard data should be displayed in company currency```

Reply:
I'll change it asap

Reply:
Does it bring value tho? Isn't it always the same?

Reply:
First case was already tested by another test which I merged with this one, I think it's worth being exhaustive, just in case, no?

Reply:
Sure, why not. (I don't see when we could break one without another but kay).